crates/rust/src/lib.rs

```diff
@@ -825,8 +843,19 @@ impl WorldGenerator for RustWasm {
         self.types.analyze(resolve);
         self.world = Some(world);
 
+        let world = &resolve.worlds[world];
+        // Specify that all imports local to the world's package should be generated
+        for (key, item) in world.imports.iter().chain(world.exports.iter()) {
+            if let WorldItem::Interface { id, .. } = item {
+                if resolve.interfaces[*id].package == world.package {
+                    let name = resolve.name_world_key(key);
+                    self.with.insert(name, InterfaceGeneration::Generate);
```

Comment:
Could this avoid overriding anything previous there if something was already specified?